crates/sui/src/client_commands.rs

```diff
@@ -638,7 +639,6 @@ impl SuiClientCommands {
 
 pub struct WalletContext {
     pub config: PersistedConfig<SuiClientConfig>,
-    pub keystore: SuiKeystore,
```

Comment:
we do not need to store the keystore in WalletContext because SuiClientConfig already has it
```
pub struct SuiClientConfig {
    pub keystore: Keystore,
    pub client_type: ClientType,
    pub active_address: Option<SuiAddress>,
}
```

this accounts for a good amount of changes in this diff

Reply:
hmm this will serialise the keystore content to the config file, also the keystore should't be serialisable